examples/experimental/point-cloud-laz-vr/app.js

```diff
@@ -0,0 +1,245 @@
+  const xMid = (xMin + xMax) / 2;
+  const yMid = (yMin + yMax) / 2;
+  const zMid = (zMin + zMax) / 2;
+
+  for (let i = 0; i < points.length; i++) {
+    points[i].position[0] = (points[i].position[0] - xMid) / scale;
+    points[i].position[1] = (points[i].position[1] - yMid) / scale;
+    points[i].position[2] = (points[i].position[2] - zMid) / scale;
+  }
+}
+
+class Example extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this._onViewportChange = this._onViewportChange.bind(this);
+    this._onInitialized = this._onInitialized.bind(this);
+    this._onResize = this._onResize.bind(this);
+    this._onUpdate = this._onUpdate.bind(this);
+
+    this.state = {
+      width: 0,
+      height: 0,
+      points: [],
+      progress: 0,
+      rotating: false,
+      vrDisplay: new EmulatedVRDisplay(),
+      emulatedPose: {
+        orientation: [0, 0, 0, 1],
+        position: [0, 0, 0]
+      }
+    };
+  }
+
+  componentWillMount() {
+    window.addEventListener('resize', this._onResize);
+    this._onResize();
+  }
+
+  componentDidMount() {
+    const {points} = this.state;
+
+    const skip = 50;
+    loadLazFile(`${DATA_REPO}/${FILE_PATH}`).then(rawData => {
+      parseLazData(rawData, skip, (decoder, progress) => {
+        for (let i = 0; i < decoder.pointsCount; i++) {
+          const {color, position} = decoder.getPoint(i);
+          points.push({color, position});
+        }
+
+        if (progress >= 1) {
+          normalize(points);
+        }
+
+        this.setState({points, progress});
+      });
+    });
+    this._initVRDisplay();
+    window.requestAnimationFrame(this._onUpdate);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this._onResize);
+  }
+
+  _onResize() {
+    const size = {width: window.innerWidth, height: window.innerHeight};
+    this.setState(size);
+    this._onViewportChange();
+  }
+
+  _onInitialized(gl) {
+    setParameters(gl, {
+      clearColor: [0.07, 0.14, 0.19, 1],
+      blendFunc: [gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA]
+    });
+  }
+
+  _onViewportChange(viewport) {
+    this.setState({
+      viewport: {...this.state.viewport, ...viewport}
+    });
+  }
+
+  _onUpdate() {
+    const {viewport} = this.state;
+    this.setState({
+      viewport: {
+        ...viewport,
+      }
+    });
+    window.requestAnimationFrame(this._onUpdate);
+  }
+
+  _renderLazPointCloudLayer() {
+    const {points} = this.state;
+    if (!points || points.length === 0) {
+      return null;
+    }
+
+    return new PointCloudLayer({
+      id: 'laz-point-cloud-layer',
+      data: points,
+      coordinateSystem: COORDINATE_SYSTEM.IDENTITY,
+      getPosition: d => d.position,
+      getNormal: d => [0, 0.5, 0.2],
+      getColor: d => [255, 255, 255, 128],
+      radiusPixels: 1
+    });
+  }
+
+  _initVRDisplay() {
+    // initialize WebVR Polyfill
+    const polyfill = new WebVRPolyfill({
+      PROVIDE_MOBILE_VRDISPLAY: true
+    });
```

Comment:
polyfill is defined but not used, I noticed the official example of webvr-polyfill does this too...

either use
```
    /* eslint-disable no-unused-vars */
    const polyfill = new WebVRPolyfill({
      PROVIDE_MOBILE_VRDISPLAY: true
    });
    /* eslint-enable no-unused-vars */
```
or
```
    const polyfill = new WebVRPolyfill({  // eslint-disable-line no-unused-vars
      PROVIDE_MOBILE_VRDISPLAY: true
    });
```
to disable the linter rule here.